website/next.config.js

```diff
@@ -8,7 +8,7 @@ dotenv.config()
 
 module.exports = withHashicorp({
   defaultLayout: true,
-  transpileModules: ['is-absolute-url', '@hashicorp/react-.*'],
+  transpileModules: ['is-absolute-url'],
```

Comment:
The updated `nextjs-scripts` has [a default setting to transpile `@hashicorp/react-*` modules](https://github.com/hashicorp/nextjs-scripts/blob/1e7f68a03f937e347c439ea0fd5c9cbfaf07c2eb/index.js#L28), so we don't need to repeat it here.